website/source/docs/guides/consul-splitting.md

```diff
@@ -0,0 +1,461 @@
+sidecar with the following command. This command will run in the foreground, so
+you’ll need to open a new terminal window after you run it.
+
+```shell
+$ docker-compose -f docker-compose-v2.yml up
+```
+
+Check that the service and its proxy have registered by checking for new `v2`
+tags next to the API service and API sidecar proxies in the Consul UI.
+
+### Configure Service Splitting - 50% Version 1, 50% Version 2
+
+Now that version 2 is running and registered, the next step is to gradually
+increase traffic to it by changing the weight of the v2 service subset in the
+service splitter configuration. Let’s increase the weight of the v2 service to
+50%. Remember; total service weight must equal 100, so you also reduce the
+weight of the v1 subset to 50. The configuration file is already in your demo
+environment at `l7_config/api_service_splitter_50_50.json` and it contains the
+following.
+
+```json
+{
+  "kind": "service-splitter",
+  "name": "api",
+  "splits": [
+    {
+      "weight": 50,
+      "service_subset": "v1"
+    },
+    {
+      "weight": 50,
+      "service_subset": "v2"
+    }
+  ]
+}
+```
+
+Apply the configuration as before.
+
+```shell
+$ curl localhost:8500/v1/config -XPUT -d @l7_config/api_service_splitter_50_50.json
+true%
+```
+
+Now that you’ve increased the percentage of traffic to v2, curl the web service
+again. Consul will equally distribute traffic across both of the service
+subsets.
+
+```hcl
+$ curl localhost:9090
+Hello World
+###Upstream Data: localhost:9091###
+  Service V1%
+$ curl localhost:9090
+Hello World
+###Upstream Data: localhost:9091###
+  Service V2%
+$ curl localhost:9090
+Hello World
+###Upstream Data: localhost:9091###
+  Service V1%
+```
+
+### Configure Service Splitting - 100% Version 2
+
+Once you are confident that the new version of the service is operating
+correctly, you can send 100% of traffic to the version 2 subset. The
+configuration for a 100% split to version 2 contains the following.
+
+```json
+{
+  "kind": "service-splitter",
+  "name": "api",
+  "splits": [
+    {
+      "weight": 0,
+      "service_subset": "v1"
+    },
+    {
+      "weight": 100,
+      "service_subset": "v2"
+    }
+  ]
+}
+```
+
+Apply it with a call to the HTTP API `config` endpoint as you did before.
+
+```shell
+$ curl localhost:8500/v1/config -XPUT -d @l7_config/api_service_splitter_0_100.json
+true%
+```
+
+Now when you curl the web service again. 100% of traffic is sent to the version
+2 subset.
+
+```hcl
+$ curl localhost:9090
+Hello World
+###Upstream Data: localhost:9091###
+  Service V2%
+$ curl localhost:9090
+Hello World
+###Upstream Data: localhost:9091###
+  Service V2%
+$ curl localhost:9090
+Hello World
+###Upstream Data: localhost:9091###
+  Service V2%
+```
+
+Typically in a production environment, you would now remove the version 1
+service to release capacity in your cluster. Congratulations, you’ve now
+completed the deployment of version 2 of your service.
```

Comment:
I saw "Clean Up" below and thought this was steps to remove the configs for splutting now we are done but it's not. Should we also include a section explaining how you would get configs back to a good state (i.e. remove the splitter again) once the v1 services are all decommissioned? We could either remove the splitter or remove the v1 branch ready for the next version release (would mean you could skip pre-creation of a splitter with 100% v2 etc.).

Also, how about calling the section below "Demo Clean Up" or something to make it clear this is not clean up you'd do in prod but just for this guide?